src/main/scala/com/cognite/sdk/scala/v1/fdm/common/properties/PropertyDefinition.scala

```diff
@@ -20,7 +25,7 @@ object PropertyDefinition {
     val `type`: PropertyType
   }
 
-  sealed trait ViewPropertyDefinition extends PropertyDefinition
+  sealed trait ViewPropertyDefinition extends PropertyDefinition with ViewPropertyCreateDefinition
```

Comment:
looks suspicious, why do we need to extend createdefinition? :eyes:

Reply:
Hmm, you are right, I will investigate if I forgot to remove this or if I can explain it

Reply:
So, the ConnectionDefinition needs to extend ViewPropertyCreateDefinition  (obviously because create view can now contain ConnnectionDefintion) but not the viewPropertyDefinition. I fixed it.